packages/tldraw/src/lib/ui/components/primitives/TldrawUiInput.tsx

```diff
@@ -74,14 +74,22 @@ export const TldrawUiInput = React.forwardRef<HTMLInputElement, TLUiInputProps>(
 				setIsFocused(true)
 				const elm = e.currentTarget as HTMLInputElement
 				rCurrentValue.current = elm.value
-				editor.timers.requestAnimationFrame(() => {
-					if (autoSelect) {
-						elm.select()
-					}
-				})
+				if (editor) {
+					editor.timers.requestAnimationFrame(() => {
+						if (autoSelect) {
+							elm.select()
+						}
+					})
+				} else {
```

Comment:
Not sure I like the `useMaybeEditor` pattern. Think I'd prefer passing in a prop called `raf` here. The only problem is how to communicate that to the end users.

Reply:
I had a raf prop there before but it was tedious. I'd rather the ugly code be in the component I guess